src/util_driver.py

```diff
@@ -0,0 +1,7 @@
+import core_util as core
```

Comment:
This did not work as `core_util` but it works as `steg_core`. Either change the filename or the import.